src/graphics/PlanningView.ts

```diff
@@ -694,6 +626,31 @@ export async function runPredictions(underworld: Underworld) {
   if (globalThis.runPredictionsPanel) {
     globalThis.runPredictionsPanel.update(Date.now() - startTime, 300);
   }
+
+  const currentlyWarningOutOfRange = warnings.has(TEXT_OUT_OF_RANGE);
+  // draw predictions
+  // Modify and draw all of the stored predictions
+  // If out of range, set color to grey
+  if (predictionGraphics && !globalThis.isHUDHidden) {
+    for (let { points, color, text } of predictionPolys) {
+      const colorOverride = currentlyWarningOutOfRange ? colors.outOfRangeGrey : color;
+      drawUIPoly(predictionGraphics, points, colorOverride, text);
+    }
+    for (let { target, color, radius, startArc, endArc, text } of predictionCones) {
+      const colorOverride = currentlyWarningOutOfRange ? colors.outOfRangeGrey : color;
+      drawUICone(predictionGraphics, target, radius, startArc, endArc, colorOverride);
+    }
+    for (let { target, color, radius, text } of predictionCircles) {
+      const colorOverride = currentlyWarningOutOfRange ? colors.outOfRangeGrey : color;
+      drawUICircle(predictionGraphics, target, radius, colorOverride, text);
+    }
+  }
+  if (globalThis.radiusGraphics) {
+    for (let { target, color, radius, text } of predictionCirclesFill) {
+      //const colorOverride = currentlyWarningOutOfRange ? colors.outOfRangeGrey : color;
```

Comment:
Prediction bloat is red instead of grey when enemy is out of range.  It should be grey to clearly denote to the user that this  _would_ be their bloat explosion radius but they're out of range and can't be cast on

Reply:
I'm not sure if this is the area where that went wrong though, just noticed that it's commented out and might be an issue here too